### PR TITLE
Update Citation and Bump Version

### PR DESCRIPTION
**Pull request by ShikharJ**

### CODE REVIEW

**Comment by adityakusupati** [comment]
lgtm. @oindrilasaha anything else that needs to be added in the readmes for RNNPool?

File: pytorch/README.md [hunk 1]
```diff
@@ -22,16 +22,19 @@ for these algorithms are in `edgeml_pytorch.trainer`.
...
+    2 layer SRNN network which can be instantiated with a choice of RNN cell. The training
     routine for SRNN is in `edgeml_pytorch.trainer.srnnTrainer`.
 5. [DROCC & DROCC-LF](https://github.com/microsoft/EdgeML/blob/master/docs/publications/drocc.pdf): `edgeml_pytorch.trainer.drocc_trainer` implements 
     a DROCC meta-trainer for training any given model architecture
-    for one-class classification on the supplied dataset. `edgeml_pytorch.trainer.drocclf_trainer` implements the DROCC-LF varaint
+    for one-class classification on the supplied dataset. `edgeml_pytorch.trainer.drocclf_trainer` implements the DROCC-LF variant
     for training models for one-class classification with limited negatives.
+6. [RNNPool](https://github.com/microsoft/EdgeML/blob/master/docs/publications/RNNPool.pdf): `edgeml_pytorch.graph.RNNPool` implements
+   the RNNPool pooling layer which can be instantiated with the dimensions of the input patch
+   and the hidden states. Currently only the inference code is implemented.
```
Comment (adityakusupati) [inline pytorch/README.md:37]: I didn't understand what the last sentence means. @oindrilasaha comments?

Comment (harsha-simhadri) [reply pytorch/README.md:37]: will resolve with oindrila and merge this PR.

Comment (ShikharJ) [reply pytorch/README.md:37]: Let's wait on the camera ready to be submitted, since I'll need to upload that here as well.

Comment (ShikharJ) [reply pytorch/README.md:37]: Okay, camera ready is uploaded now. Please feel free to resolve and merge.

**Review by harsha-simhadri — APPROVED** [review]

**Pull request merged**